go.mod

```diff
@@ -1603,3 +1611,6 @@ replace (
 	go.mongodb.org/mongo-driver => go.mongodb.org/mongo-driver v1.9.4
 	google.golang.org/grpc => google.golang.org/grpc v1.56.3
 )
+
+// go-otel-audit pins
```

Comment:
Please explain why this is necessary - it's clear that the `replace` directive is pinning :)